.github/workflows/build_wheels_windows.yml

```diff
@@ -135,6 +136,9 @@ jobs:
           BUILD_PARAMS: ${{ inputs.wheel-build-params }}
         run: |
           source "${BUILD_ENV_FILE}"
+          export FFMPEG_ROOT="${PWD}"/third_party/ffmpeg
```

Comment:
nit: We now ask users to define these env vars in an env var script (https://github.com/pytorch/test-infra/blob/main/.github/workflows/build_wheels_linux.yml#L38). We can do this in a follow-up

Reply:
Yes will post a follow-up pr, to refactor this with linux and macos jobs as well, linux is doing same thing now:
https://github.com/pytorch/test-infra/blob/main/.github/workflows/build_wheels_linux.yml#L138